### PR TITLE
Add TRG 4.01 - Image Tagging

### PR DESCRIPTION
**Pull request by carslen**
Initial release of TRG 4.01 - Image Tagging

### CODE REVIEW

File: docs/trg/trg-4/trg-4-1.md [hunk 1]
```diff
@@ -1,16 +1,70 @@
...
-|----------------------|--------|--------------|--------------|
-| Catena-X System Team | Draft  | 13-Sept-2022 | n/a          |
+| Author               | Status | Created     | Post-History     |
+|----------------------|--------|-------------|------------------|
+| Catena-X System Team | Active | 10-Nov-2022 | Initital release |
 
 ## Description
 
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore.
+If a container image is part of a Tractus-X product, the image must be build/tagged as follows:
```
Comment (almadigabor) [inline docs/trg/trg-4/trg-4-1.md:11]:
```suggestion
If a container image is part of a Tractus-X product, the image must be built/tagged as follows:
```

File: docs/trg/trg-4/trg-4-1.md [hunk 2]
```diff
@@ -1,16 +1,70 @@
...
+
+```shell
+docker build -t image:SemVer -t image:latest .
+```
+
+After building a new container image, the image must be tagged with `SemVer` and `latest`.
```
Comment (almadigabor) [inline docs/trg/trg-4/trg-4-1.md:17]:
```suggestion
After building a new container image, the image must be tagged with [SemVer](https://semver.org/) and `latest`.
```

File: docs/trg/trg-4/trg-4-1.md [hunk 3]
```diff
@@ -1,16 +1,70 @@
...
 
 ## Why
 
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore
-magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo.
+The main reason for this TRG is to avoid the usage of `latest` for image tagging.
+
+As these container images will usually be used in versioned Helm charts, using `latest` as tag would destroy the idea of
+a versioned and released Helm chart, as a changed image could be injected without changing the Helm chart version. So a
+specific Helm chart version could behave different, due to the underlying container image. This behavior is unwanted.
```
Comment (almadigabor) [inline docs/trg/trg-4/trg-4-1.md:25]:
```suggestion
specific Helm chart version could behave differently, due to the underlying container image. This behavior is unwanted.
```

**Review by almadigabor — COMMENTED** [review]
Please see suggestions

**Comment by carslen** [comment]
Article got a little rewrite. Included your suggestions.

**Pull request merged**